src/ophys_etl/workflows/ophys_experiment.py

```diff
@@ -85,3 +91,114 @@ def from_id(
             session=session,
             specimen=specimen
         )
+
+    def get_ophys_experiment_motion_border(self,
```

Comment:
See comment about converting this to a property and creating `session` rather than passing it.